server_environment/serv_config.py

```diff
@@ -29,8 +30,15 @@
 
 from .system_info import get_server_environment
 
-from odoo.addons import server_environment_files
-_dir = os.path.dirname(server_environment_files.__file__)
+_logger = logging.getLogger(__name__)
+
+try:
+    from odoo.addons import server_environment_files
+    _dir = os.path.dirname(server_environment_files.__file__)
```

Comment:
What about using 
`ck_path = odoo.modules.module.get_resource_path('server_environment_files', system_base_config['running_env'])`?

Another question, if I'm not using this module but it is declared then will raise an error?

Reply:
@moylop260 Thank you for the review. The change is a backport of a commit from 11.0 (OCA/server-env@d9ad47f)
> What about using
`ck_path = odoo.modules.module.get_resource_path('server_environment_files', system_base_config['running_env'])`?


`system_base_config['running_env']`will fail since the section is only required when you use server_environment_files.


> Another question, if I'm not using this module but it is declared in the addons-path then will raise an error?

IMO Yes it will raise an error.
https://travis-ci.org/OCA/server-auth/jobs/454436281#L747